assets/src/edit-story/utils/processPastedNodeList.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import escapeHTML from './escapeHTML';
+
+const ALLOWED_CONTENT_NODES = ['strong', 'em', 'u'];
```

Comment:
We don't want to keep links? I assume that's just a temporary thing, because when we in the future support inline links (in #1158), we'll support pasted links too.

Reply:
Also, what if the content comes with `<b>` or `<i>` tags? They ought to be converted to `<strong>` or `<em>` respectively.

Reply:
We might want to keep the links in the future, however, this Basic Text paste PR was meant to be very basic :) -- to get to alpha on time and have basic pasting working.

I'd defer most of the improvements to a future PR (#760) where the more detailed behavior should get confirmed by UX as well (e.g. what to do with links and headings and if there's an image in the middle of the text etc.)

Good point about `b` and `i`! Will check.

Reply:
Sure, let's leave it for the future. I think nested tags are important for alpha though - more than `<b>` and `<i>` really.